specs/middleware-injector.specs.ts

```diff
@@ -52,6 +59,20 @@ describe("middleware-injector", () => {
     assert.equal(newContentSource, (sourceCode + oldContentSource));
   });
 
+  it("Should find the content script entries and inject the middleware source on them in case CS as array", () => {
```

Comment:
You can just turn the above test (line 50) into a describe and create 2 "it" , one case the content script being a string and the another case as a array.
As they're within the same context, but just with a small difference, makes more sense both being inside a same "describe".
Another point, you're not testing the background, but as the test is also a specification, it must **contain it**. Don't be afraid to create a common function to test the missing background part (as in line 53) and call in both tests, this avoid code duplications (**DRY**), and you maintain the specification correct.